src/link/api.js

```diff
@@ -1,35 +1,49 @@
 let abortFunctions = [];
 
-export const prefixSearch = ( lang, term, callback ) => {
-	const url = `https://${ lang }.wikipedia.org/w/rest.php/v1/search/title?q=${ term.trim() }&limit=5`;
-	return request( url, ( data ) => {
-		if ( ! data.pages ) {
+const handler = ( callback ) => {
+	return ( data ) => {
+		if ( ! data.query || ! data.query.pages ) {
```

Comment:
I see the old code used the optional chaining operator `?.`. Any reason to not use it here?

Reply:
It's not something we use often so I wasn't sure about browser support. I made a mental note of looking into it.